scripts/deploy.sh

```diff
@@ -1,65 +0,0 @@
-#!/bin/bash
```

Comment:
This is required for deployments – https://github.com/alphagov/datagovuk_publish/blob/6c7654d0d0f646ba5bc5f859cfe58978b25b82c6/.travis.yml#L33